### ISSUE TITLE
matlab-cell, rename to matlab sections

### ISSUE THREAD

**Issue by JohnC32** [mathworks/Emacs-MATLAB-Mode#5]
MATLAB cells are things like {1, 2, 3}.

At one point in the past MathWorks all called the "%% sections" in *.m files cells, but this terminology no longer exists. MathWorks now uses sections of the m-file, see https://www.mathworks.com/help/matlab/matlab_prog/create-and-run-sections.html

Therefore, it would be good to switch from matlab-cell's to matlab-code-sections.el

**Comment by ouboub** [comment]
>>> "JC" == John Ciolfi ***@***.***> writes:

> MATLAB cells are things like {1, 2, 3}.
> At one point in the past MathWorks all called the "%% sections" in *.m
> files cells, but this terminology no longer exists. MathWorks now uses
> sections of the m-file, see
> https://www.mathworks.com/help/matlab/matlab_prog/create-and-run-sections.html

> Therefore, it would be good to switch from matlab-cell's to matlab-code-sections.el


Nidish is on it, but then also the command 
 command matlab-shell-run-cell

Should be renamed.

-- 
I strongly condemn Hamas heinous despicable pogroms/atrocities on Israel
I strongly condemn Putin's war of aggression against Ukraine.
I support to deliver weapons to Ukraine's military. 
I support the EU and NATO membership of Ukraine.

**Comment by Nidish96** [comment]
I had a brief email exchange with Uwe about this since I'm personally just too used to referring to these as cells. 
I'm not opposed to renaming. 

Renaming, however, requires a more extensive change since the term cell does not only occur in matlab-cell.el. It occurs in matlab-shell.el also, for instance (matlab-shell-run-cell). This may cause confusions to other devs if done very abruptly.
I personally don't have too much experience in maintaining so I'll defer this decision to you guys. 

I've created a pull request from the matlab-cells branch for now. Everything in matlab-cell.el (including the name) has been converted to using the matlab-sections terminology and I have verified them to work.
